src/argilla/client/feedback/dataset/base.py

```diff
@@ -376,11 +404,15 @@ def prepare_for_training(
                     lang = spacy.blank(lang)
```

Comment:
```suggestion
                    lang = spacy.blank(lang) if len(lang)==2 else spacy.load(lang)
```

Reply:
FYI this suggestions does not really work ou-of-the box😅

Reply:
I don't think I understand, this line is already wrapped in
```python
                if len(lang) == 2:
                    lang = spacy.blank(lang)
                else:
                    lang = spacy.load(lang)
```